compiler_gym/envs/llvm/service/LlvmSession.cc

```diff
@@ -129,7 +129,10 @@ Status LlvmSession::applyAction(const Action& action, bool& endOfEpisode,
   switch (actionSpace()) {
     case LlvmActionSpace::PASSES_ALL:
       LlvmAction actionEnum;
-      RETURN_IF_ERROR(util::intToEnum(action.action(), &actionEnum));
+      if (action.choice_size() != 1) {
```

Comment:
should the condition be `< 1`? 
Otherwise the error message `"Missing choice"` is misleading

Reply:
Thanks, I've improved the error message